File: public/phase-2-20160620022203/JAVA/alcohol-calculator.java
// Name : Alcohol Calculator
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n1 : alcohol percentage
		// m1 : volume in ml
		// n2 : alcohol percentage
		// m2 : volume in ml
				Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int n1 = Integer.parseInt(elements[0]);
		float m1 = Float.parseFloat(elements[1]);
		String[] elements = (in.nextLine()).split(" ");
		int n2 = Integer.parseInt(elements[0]);
		float m2 = Float.parseFloat(elements[1]);


		// write your code here
		// store your results in `v`

		// output
		// Dummy Data
		String v="30%";

		System.out.println(v);

	}
}

File: public/phase-2-20160620022203/C/alcohol-calculator.c
// Name : Alcohol Calculator
#include <stdio.h>
int main() {
	// input:
	// n1 :  alcohol percentage 1
	// m1 : volume 1 in ml
	// n2 :  alcohol percentage 2
	// m2 : volume 2 in ml
	int n1, n2;
	float m1, m2;
	char v[105]="30%";
	scanf("%d %f", &n1, &m1);
	scanf("%d %f", &n2, &m2);


	// write your code here
	// store your results in `v`

	// output
	printf("%s", v);
	return 0;
}